StereoKitC/xr_backends/extensions/hand_tracking.cpp~RF1c29c2ca.TMP

```diff
@@ -0,0 +1,123 @@
+/* SPDX-License-Identifier: MIT */
```

Comment:
The filename here looks like a temp file was included 🤔 it looks to have different content than the non-temp version, so worth double checking which one is "right"!